src/Webflow/Api.php

```diff
@@ -84,12 +95,12 @@ private function delete($path)
     private function parse($response)
     {
         $json = json_decode($response);
-        if (isset($json->code) && isset($json->msg)) {
-            $error = $json->msg;
-            if (isset($json->problems)) {
-                $error .= PHP_EOL . implode(PHP_EOL, $json->problems);
+        if (isset($json->code) && isset($json->message)) {
+            $error = $json->message;
+            if (isset($json->details)) {
+                $error .= PHP_EOL . $json->code . ': ' . implode(PHP_EOL, $json->details) ;
```

Comment:
I would like to ensure that implode(PHP_EOL, $json->details) will work when details has info.  I didnt get these errors.